### PR TITLE
More refined on-boarding messaging.

### PR DESCRIPTION
**Pull request by pniko**
Cleaning up the messaging in the SDK so we can update the Getting Started page in the portal.

### CODE REVIEW

**Comment by guperrot** [comment]
CI error was:

```

File: appcenter-push/scripts/postlink.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 const rnpmlink = require('appcenter-link-scripts');
 
+console.log("Please enter your Android and iOS app secrets below.");
```
Comment (dhei) [inline appcenter-push/scripts/postlink.js]: please use single quotes ;)

File: appcenter-link-scripts/src/android/index.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = {
         }
         return inquirer.prompt([{
             type: 'input',
-            message: 'What is the Android App Secret?',
+            message: 'What secret does your Android app use? [None]',
```
Comment (dhei) [inline appcenter-link-scripts/src/android/index.js]: @pniko Question, so user has to delete `[None]` and then enter the app secret in the prompt?

Comment (guperrot) [reply appcenter-link-scripts/src/android/index.js]: He is just changing the message, there is still no default value.

Comment (pniko) [reply appcenter-link-scripts/src/android/index.js]: It's just to signal you can hit return. We are updating the Getting Started pages to stop directing users to flip to another app for the other secret. Narrowing the Getting Started instructions to the platform they are staring at makes things a bit cleaner until we let users group apps together in the future.

**Pull request merged**